src/components/ProjectsMenu.vue

```diff
@@ -78,4 +84,12 @@ export default {
     }
   }
 };
-</script>
\ No newline at end of file
+</script>
+<style scoped>
+  .project-menu-too-long-description {
+    display: -webkit-box;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
```

Comment:
@volterra79 Please, we can live even better without this vendor specific rules...

**See:** https://stackoverflow.com/questions/15662578/flexible-box-model-display-flex-box-flexbox

Reply:
@Raruto i'll check

Reply:
@Raruto changed. Please check and let me know. Thanks